fractal_tasks_core/cellpose_segmentation.py

```diff
@@ -167,6 +384,8 @@ def cellpose_segmentation(
     min_size: int = 15,
     augment: bool = False,
     net_avg: bool = False,
+    use_masks: bool = False,
```

Comment:
To be discussed: Shouldn't the default for this task be that masking is used IF the ROI table contains masks?

Reply:
I'd be in favor of having a `use_masks` optional input that defaults to True. Then, the task checks whether the provided ROIs have a mask (ROI_mask_present).
Then:
```
use_masks = use_masks and ROI_mask_present
``` 

=> One can manually turn off use_masks, but it's on by default if the ROI table provided by the user has masking information.

What counts as masking information? The ROI table having a labels column? Or do we have a better readout? Could also use the .zattrs pointing to a label image with their `region` param + the `instance_key` being defined & present in obs

Reply:
Agreed. As discussed, the task now defaults to `use_masks=True`. This means that (unless specified), it will always try to proceed with masked loading.
If the table does not satisfy the criteria defined in `lib_regions_of_interest.is_ROI_table_valid`, then the task will set `use_masks=False`.